Userland/Libraries/LibWeb/DOM/Document.h

```diff
@@ -198,8 +213,8 @@ class Document
     Color background_color() const;
     Vector<CSS::BackgroundLayerData> const* background_layers() const;
 
-    Color link_color() const;
-    void set_link_color(Color);
+    Color normal_link_color() const;
```

Comment:
I am no maintainer and I know very little about `LibWeb`, but in general it's nice to keep commits atomic.
Here that would be, first renaming `link_color` in one commit and then add your changes to a second one.